api/envoy/extensions/filters/network/meta_protocol_proxy/v3/meta_protocol_proxy.proto

```diff
@@ -0,0 +1,80 @@
+
+syntax = "proto3";
+
+package envoy.extensions.filters.network.meta_protocol_proxy.v3;
+
+import "envoy/config/core/v3/config_source.proto";
+import "envoy/extensions/filters/network/meta_protocol_proxy/v3/route.proto";
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.network.meta_protocol_proxy.v3";
+option java_outer_classname = "MetaProtocolProxyProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// [#protodoc-title: Meta Protocol proxy]
+// Meta Protocol proxy.
+
+// [#not-implemented-hide:]
+// [#next-free-field: 7]
+message MetaProtocolProxy {
+  // The human readable prefix to use when emitting statistics.
+  string stat_prefix = 1 [(validate.rules).string = {min_len: 1}];
+
+  // The name of the application protocol built on top of meta protocol.
+  string application_protocol = 2 [(validate.rules).string = {min_len: 1}];
+
+  oneof route_specifier {
+    option (validate.required) = true;
+
+    // The meta protocol proxy’s route table will be dynamically loaded via the RDS API.
+    Rds rds = 3;
```

Comment:
This is the same discussion we had in https://github.com/envoyproxy/envoy/pull/17631 which led to TRDS; if we are reusing `Rds` here, we should be shipping the `RouteConfiguration` as it exists today. Otherwise we need a MetadataRDS. CC @markdroth

Reply:
@htuch per what we disscussed in the design docs, it's been renamed as MetaRds.